### DISCUSSION TITLE
Why Immich create a folder called library ?

### DISCUSSION THREAD

**Discussion by Narkowii** [immich-app/immich#7905] [Q&A]
Hello, 

Why Immich create a folder called library when i bacup my photo's phone ??

I mount a share on my NAS, and Immich put my phone photos on my NAS => My mount share/library/myaccount/

Is it possible to configure something to have on my NAS =>My mount share/myaccount/

thanks !

**Comment by schuhbacca** [comment]
You can read up here about immich's storage system and ask any questions after: https://immich.app/docs/administration/backup-and-restore#filesystem

😄
